adviser/base/openshift-templates/dependency-monkey.yaml

```diff
@@ -45,43 +42,34 @@ parameters:
     displayName: Beam width
     value: "1000000"
   - name: THOTH_DEPENDENCY_MONKEY_STACK_OUTPUT
-    required: false
```

Comment:
why do we remove them?

Reply:
argocd fails to deploy as it sees this variable as default if set to false. 
showed this issue during the demo as well.

Reply:
You explained it today very well today indeed :) Thanks!! Yesterday I was still ignorant :D